src/components/Table/GridTable.tsx

```diff
@@ -437,21 +458,52 @@ function renderCssGrid<R extends Kinded>(
   headerRows: RowTuple<R>[],
   filteredRows: RowTuple<R>[],
   firstRowMessage: string | undefined,
-  stickyHeader: boolean,
+  _stickyHeader: boolean,
   firstLastColumnWidth: number | undefined,
   xss: any,
-  virtuosoRef: MutableRefObject<VirtuosoHandle | null>,
+  _virtuosoRef: MutableRefObject<VirtuosoHandle | null>,
 ): ReactElement {
+  // We must determine if the header is using nested card styles to account for
+  // the opening and closing Chrome rows.
+  const isNestedCardStyleHeader = !!style.nestedCards?.kinds["header"];
+  /*
+    Determine at what CSS selector index is the first non header row. Note that
+    CSS selectors are not zero-based, unlike JavaScript, so we must add 1 to
+    what we'd normally expect.
+
+    Ex: When we don't have nestedCard styled headers, then we don't have opening
+    and closing Chrome rows. Therefore, the first non-header row is the second
+    row and since CSS selectors are not zero-based, we are aiming for a value
+    of 2 (1 + 1).
+
+    Ex: When we have nestedCard styled headers, then we have opening and closing
+    Chrome rows. Therefore, the first non-header row is the fourth row because:
+    - Row 1 is the opening Chrome Row
+    - Row 2 is the header
+    - Row 3 is the closing Chrome Row
+    - Row 4 is the first non-header row
+    And since CSS selectors are not zero-based, we are aiming for a value of 4
+    (3 + 1).
+  */
+  const firstNonHeaderRowIndex = (!isNestedCardStyleHeader ? 1 : 3) + 1;
+
   return (
     <div
       css={{
         ...Css.dg.gtc(calcDivGridColumns(columns, firstLastColumnWidth)).$,
-        // Apply the between-row styling with `div + div > *` so that we don't have to have conditional
-        // `if !lastRow add border` CSS applied via JS that would mean the row can't be React.memo'd.
-        // The `div + div` is also the "owl operator", i.e. don't apply to the 1st row.
-        ...(style.betweenRowsCss ? Css.addIn("& > div + div > *", style.betweenRowsCss).$ : {}),
-        // removes border between header and second row
-        ...(style.firstNonHeaderRowCss ? Css.addIn("& > div:nth-of-type(2) > *", style.firstNonHeaderRowCss).$ : {}),
+        /*
+          Using n + (firstNonHeaderRowIndex + 1) here to target all rows that
+          are after the first non-header row. Since n starts at 0, we can use
+          the + operator as an offset.
+
+          Inspired by: https://stackoverflow.com/a/25005740/2551333
+        */
+        ...(style.betweenRowsCss
+          ? Css.addIn(`& > div:nth-of-type(n + ${firstNonHeaderRowIndex + 1}) > *`, style.betweenRowsCss).$
```

Comment:
> here to target all rows that are after the first non-header row

Seems like this is going to apply `betweenRowsCss` to both regular rows as well as chrome rows?

Granted, it was doing that before...but I don't know that that was on purpose...

If I'm following, it seems like you've fixed not apply `betweenRowCss` and `firstNonheaderRowCss` to chrome rows within the header, but we've not yet fixed it (or even decided what it should really do?) for non-header regular vs. chrome rows?

Reply:
I kinda wonder if, when using nested cards, `betweenRowCss` should just target the chrome/space rows? I think we've got a `data-` attribute we could use.

Reply:
Yeah, this is a can of worms that I did not intend to touch but in that case, where we only target Chrome, spacers, this will get pretty complicated and weird for the user trying to understand what it does. 

This seems to be more related to a non-nested approach. Maybe even renaming it as @bdow suggested since the only usage we can see is adding a border-top style.

Reply:
Huh, that is an interesting point... you are right it's only used for border-tops, so maybe we could "dumb down" / remove `betweenRowCss` insofar as turning into a "betweenRowBorder" that would only a) have border color + size and b) be for non-card tables?

I.e. for card tables I think we assume the cards are serving the UX function of row delineation, so why bother?

Fwiw if we wanted to simplify all of this by just throwing a runtime error when both cards and `betweenRowCss` / `betweenRowBorder` are used, and sneak that into this PR, that'd be great; also fine as a todo/cleanup.